.gitignore

```diff
@@ -110,3 +111,6 @@ mocks/mock-bie-kafka/kafka.keystore.jks
 mocks/mock-bie-kafka/kafka.truststore.jks
 svc-bie-kafka/keystore.p12
 svc-bie-kafka/truststore.p12
+tom-bip.sh
```

Comment:
is this a local filename that is being git ignored?

Reply:
yes, it should be handled via the correct git config settings.  if prefered I could definitely revert the ignore file change and just untrack that file on my local.

Reply:
Yea, that works.